src/api.ts

```diff
@@ -1110,3 +1110,6 @@ export interface ApiGatewayResponseOverride {
    */
   status: number;
 }
+
+// to prevent the closure serializer from trying to import all of functionless.
+export const deploymentOnlyModule = true;
```

Comment:
Hmm this seems hacky. Can we use a comment instead? Are we sure it's necessary. Wouod really rather not have to scatter this around

Reply:
This works with the closure serializer right now. I don't think the closure serializer can see comments. I wanted something that works and then will find ways to make it less verbose.

Reply:
What's the root cause for the over-inclusion of functionless dependencies?

Reply:
Is `deploymentOnlyModule` a feature enabled by the closure serializer or something you added ust now?

Reply:
nit: If we can control the name, I would prefer `synthTimeOnlyModule`.